File: ios/Jazzify/BalloonRush/BalloonRushMap.swift
import CoreGraphics

/// 風船ラッシュのプレイフィールド（`SurvivalMap` の 1/4）。Web `BALLOON_RUSH_MAP_CONFIG` と同期。
enum BalloonRushMap {
    static let width: CGFloat = SurvivalMap.width / 4
    static let height: CGFloat = SurvivalMap.height / 4
}

/// `public/Drums160Loop.mp3` の CDN URL（`scripts/upload-drums160-loop-r2.mjs`）。
enum BalloonRushDrumLoopBgm {
    static let urlString = "https://jazzify-cdn.com/fantasy-bgm/survival-composite-phrases-drums160-loop.mp3"
}

File: src/utils/balloonRushMap.ts
import { MAP_CONFIG } from '@/components/survival/SurvivalTypes';

/** 風船ラッシュのプレイフィールド（サバイバルマップの 1/2）。iOS `BalloonRushMap` と同期。 */
export const BALLOON_RUSH_MAP_CONFIG = {
  width: MAP_CONFIG.width / 2,
  height: MAP_CONFIG.height / 2,
  tileSize: MAP_CONFIG.tileSize,
} as const;

/** `public/Drums160Loop.mp3` → R2（`scripts/upload-drums160-loop-r2.mjs`） */
export const BALLOON_RUSH_DRUM_LOOP_BGM_URL =
  'https://jazzify-cdn.com/fantasy-bgm/survival-composite-phrases-drums160-loop.mp3';
